examples/vae_text/vae_train.py

```diff
@@ -408,34 +401,25 @@ def _generate(start_tokens: torch.LongTensor,
                 fc_output, (-1, ) + model.decoder_initial_state_size)
```

Comment:
Have you realized that the logic for computing `decoder_states` is exactly the same as in `forward`? Can we move it to `decode`?

Reply:
Yes, got it.